PKGBUILD

```diff
@@ -2,13 +2,13 @@ pkgdesc="ROS - qt_gui_cpp provides the foundation for C++-bindings for qt_gui an
 url='https://wiki.ros.org/qt_gui_cpp'
 
 pkgname='ros-melodic-qt-gui-cpp'
-pkgver='0.3.16'
+pkgver='0.4.0'
 arch=('i686' 'x86_64' 'aarch64' 'armv7h' 'armv6h')
 pkgrel=1
 license=('BSD')
 
 ros_makedepends=(
-	ros-melodic-python-qt-binding
+	'ros-melodic-python-qt-binding>=0.4.0'
```

Comment:
Can we remove the versioned dep here? There is no need for it, since we will only be shipping the latest releases anyway.

Reply:
Done.